AuxTel/AuxTel-Non-standard-Operations/AuxTel-EUI-Access/AuxTel-EUI-Access.rst

```diff
@@ -0,0 +1,107 @@
+.. This is a template for an informative/general use document. 
+
+.. Review the README in this document's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: Kshitija Kelkar
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *List-of-contributors*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Templates-Informative-Document:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. role::  raw-html(raw)
+    :format: html
+
+####################
+AuxTel EUI Access
+####################
+
+.. note::
+    This is a document template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _Informative-Document-Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the document's purpose and utilization. 
+
+.. _Informative-Document-Main-Information-Section:
+
+This document explains how to to access the AuxTel EUI through a remote desktop.
+
+AuxTel EUI Access
+========================
+
+.. todo::
+   Include utilization of admonishments (caution, warning, etc.)
+
+
+
+ 
+
+.. _Auxtel-EUI-Access-Setup-Section-one:
```

Comment:
I think adding the full name of the title of the section would be better, rather than "Section one". For instance, 
AuxTel-EUI-Access-Setup-Remote-Desktop?